src/scenes/Wallet/components/Balances/index.jsx

```diff
@@ -232,7 +227,7 @@ class MyBalances extends React.PureComponent {
     );
   }
 
-  handleSorting = (property) => (event) => {
+  handleSorting = (property) => (event) => { // eslint-disable-line
```

Comment:
I put this `eslint-disable-line` because the `event` variable is nice for readability so people know what's going on here